code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -169,38 +169,11 @@
 		..()
 
 /mob/living/carbon/human/adjustCloneLoss(amount)
-	..()
-
 	if(species.species_flags & (IS_SYNTHETIC|NO_SCAN|ROBOTIC_LIMBS))
 		cloneloss = 0
 		return
-
-	var/heal_prob = max(0, 80 - getCloneLoss())
-	var/mut_prob = min(80, getCloneLoss()+10)
-	if (amount > 0)
-		if (prob(mut_prob))
-			var/list/datum/limb/candidates = list()
-			for (var/datum/limb/O in limbs)
-				if(O.limb_status & (LIMB_ROBOT|LIMB_DESTROYED|LIMB_MUTATED)) continue
-				candidates |= O
-			if (candidates.len)
-				var/datum/limb/O = pick(candidates)
-				O.mutate()
-				to_chat(src, "<span class = 'notice'>Something is not right with your [O.display_name]...</span>")
-				return
 	else
-		if (prob(heal_prob))
-			for (var/datum/limb/O in limbs)
-				if (O.limb_status & LIMB_MUTATED)
-					O.unmutate()
-					to_chat(src, "<span class = 'notice'>Your [O.display_name] is shaped normally again.</span>")
-					return
-
-	if (getCloneLoss() < 1)
-		for (var/datum/limb/O in limbs)
-			if (O.limb_status & LIMB_MUTATED)
-				O.unmutate()
-				to_chat(src, "<span class = 'notice'>Your [O.display_name] is shaped normally again.</span>")
+		..()
```

Comment:
Return ..()? also else is not needed here

Reply:
Fixed